Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
   "libursa",
-  "libzmix"
+  "libzmix",
+  "libzmix/bulletproofs_amcl"
```

Comment:
Why do we need another workspace?

Reply:
So that this feature can be developed independently and merged back when more changes are done.